src/Octonion.jl

```diff
@@ -34,10 +33,27 @@ promote_rule(::Type{Octonion{T}}, ::Type{Quaternion{S}}) where {T <: Real, S <:
 promote_rule(::Type{Octonion{T}}, ::Type{Octonion{S}}) where {T <: Real, S <: Real} = Octonion{promote_type(T, S)}
 
 octo(p, v1, v2, v3, v4, v5, v6, v7) = Octonion(p, v1, v2, v3, v4, v5, v6, v7)
-octo(p, v1, v2, v3, v4, v5, v6, v7, n) = Octonion(p, v1, v2, v3, v4, v5, v6, v7, n)
 octo(x) = Octonion(x)
 octo(s, a) = Octonion(s, a)
 
+Base.@deprecate(
+    Octonion{T}(s::Real, v1::Real, v2::Real, v3::Real, v4::Real, v5::Real, v6::Real, v7::Real, norm::Bool) where {T<:Real},
+    Octonion{T}(s, v1, v2, v3, v4, v5, v6, v7)
+)
+Base.@deprecate(
+    Octonion(s::Real, v1::Real, v2::Real, v3::Real, v4::Real, v5::Real, v6::Real, v7::Real, norm::Bool),
+    Octonion(s, v1, v2, v3, v4, v5, v6, v7)
+)
+Base.@deprecate octo(p, v1, v2, v3, v4, v5, v6, v7, n) octo(p, v1, v2, v3, v4, v5, v6, v7)
+
+function Base.getproperty(o::Octonion, k::Symbol)
+  if k === :norm
+      Base.depwarn("`o.norm` is deprecated. Please use `isunit(o)` instead.", Symbol("o.norm"))
+      return isunit(o)
+  end
+  return getfield(o, k)
+end
```

Comment:
We already have 4 spaces indents in Octonion.jl. (e.g. https://github.com/JuliaGeometry/Quaternions.jl/blob/v0.5.6/src/Octonion.jl#L104-L114)
I think it would be better if the new code has 4 space indents.

Reply:
We have removed Octonion from this package.